vignettes/phyloVision.Rmd

```diff
@@ -0,0 +1,145 @@
+---
+title: "Introduction to PhyloVision and Hotspot"
+package: "`r BiocStyle::pkg_ver('BiocStyle')`"
+output: BiocStyle::html_document
+vignette: >
+  %\VignetteIndexEntry{Introduction to PhyloVision and Hotspot}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Preliminaries
+
+If you have yet to install VISION, we recommend installing the package from Github to install this package. Full source code can be found at the VISION Github repository, available [here](http://www.github.com/YosefLab/VISION).
```

Comment:
Since below we show users how to use Hotspot, let's also include instructions on how to install Hotspot in the preliminaries.

Reply:
I'd like to expand a bit more on Hotspot, and why we're using it. Can you also please link the website for Hotspot?